pixart.js

```diff
@@ -1 +1,45 @@
+$(document).ready(function(){
+  var color;
+  $("#set-color").on("click",function(evt){
+    evt.preventDefault();
+    setColor();
+  });
 
+  // had to comment this out to prevent setColor() from firing twice
+  // input default behavior?
```

Comment:
would you like to work through this together? It's not obvious why this would fire twice.